README.md

```diff
@@ -60,6 +60,10 @@ browserstack.username=<browserstack_username>
 browserstack.password=<browserstack_password>
 data.location=develop
 api.keys=<api_key>
+app.token.url=https://login.microsoftonline.com/<azure_app_id>/oauth2/v2.0/token
```

Comment:
Does it need to be compatible with /v1 too?
If it is the case i wonder if we can make the method using this url flexible enough so we could have both
https://login.microsoftonline.com/<azure_app_id>/oauth2/v2.0/token (v2)
https://login.microsoftonline.com/<azure_app_id>/oauth2/token (v1)

What do you think? it would be a way also to test that our custom authoriser accept both ?

Below is the code in our authoriser (and links for more details):
```js
// https://github.com/dvsa/cvs-svc-authoriser/blob/develop/src/services/signature-check.ts 


// validating our token here
export const checkSignature = async (encodedToken: string, decodedToken: any): Promise<void> => {
  // tid = tenant ID, kid = key ID
  const certificate = await getCertificateChain(decodedToken.payload.tid, decodedToken.header.kid);

  JWT.verify(
    encodedToken,
    certificate,
    {
      audience: decodedToken.payload.aud,
      issuer: decodedToken.payload.iss,
      algorithms: [ "RS256" ]
    }
  );
}

// getting azure certificate to verify our token later
export const getCertificateChain = async (tenantId: string, keyId: string): Promise<string> => {
  const keys: Map<string, string> = await getKeys(tenantId);

  const certificateChain = keys.get(keyId);

  if (!certificateChain) {
    throw new Error(`no public key with ID '${keyId}' under tenant ${tenantId}`);
  }

  return certificateChain;
}

const getKeys = async (tenantId: string): Promise<Map<string, string>> => {
  const response = await axios.get(`https://login.microsoftonline.com/${tenantId}/discovery/keys`);

  const map: Map<string, string> = new Map();

  for (const key of response.data.keys) {
    const keyId = key.kid;
    const certificateChain = `-----BEGIN CERTIFICATE-----\n${key.x5c[0]}\n-----END CERTIFICATE-----`;

    map.set(keyId, certificateChain);
  }

  return map;
}


```